### PR TITLE
[releng] Update MWE to 1.6.0/2.12.0

### PR DESCRIPTION
**Pull request by genie-xtext**
This change was brought to you by your friendly Xtext Genie.

### CODE REVIEW

File: releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="2.12.0.v20201119-2018" encoding="UTF-8"?>
```
Comment (cdietrich) [inline releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml:1]: ?????

**Review by cdietrich — APPROVED** [review]
xml version needs to be fixed

**Review by cdietrich — CHANGES_REQUESTED** [review]
xml version needs fix

**Review by cdietrich — APPROVED** [review]
manually fixed the xml version

**Review by kthoms — APPROVED** [review]

**Pull request merged**